dateparser/languages/loader.py

```diff
@@ -34,10 +36,14 @@ def _load_data(self):
             data = get_data('data', 'languages.yaml')
         else:
             data = self.file.read()
-        data = load_yaml(data)
-        base_data = data.pop('base', {})
+        
+        if not self._raw_data:
```

Comment:
We would still load data from yaml with `get_data` or `self.file.read()`. Though we don't call entire `_load_data` method if `_data` attribute is not None. I am really not sure what are you trying to achieve with `_raw_data`